tests/test_vpc.rb

```diff
@@ -87,7 +88,7 @@ def test_auto_recovery_reload_delay
     assert_equal(300, @vpc.auto_recovery_reload_delay,
                  'Auto recovery delay should be 300')
     # negative high range
-    e = assert_raises(CliError) { @vpc.auto_recovery_reload_delay = 3601 }
+    e = assert_raises(CliError) { @vpc.auto_recovery_reload_delay = 86401 }
```

Comment:
The max range for 7k has changed to 86400 in the new image I was testing. So it was not generating the cliError as expected.